### ISSUE TITLE
Navigator "Launch" of Spider fails

### ISSUE THREAD

**Issue by Bejoscha** [ContinuumIO/anaconda-issues#6301]
Operating system: win7 64bit (Professional)

After install of Anaconda, launching any of the pre-installed apps (f.e. Spyder) via the Anachonda Navigator GUI fails.
It starts promisingly with the progress bar (for several seconds) but then nothing happens.
However, starting Spyder via the anaconda console or the Windows Start menu works.

For additional information see discussion thread at: https://groups.google.com/a/continuum.io/forum/#!topic/anaconda/rLKpVFTJXCg

**Comment by goanpeca** [comment]
@Bejoscha could you open the `Anaconda Prompt` and type `$ anaconda-navigator` and when launching `Spyder` could you see if any traceback appears on the prompt?

**Comment by Bejoscha** [comment]
Hi, okay, I did:
- Windows START menu "Anaconda prompt".
- Launched "anaconda navigator" from the prompt.  (Rather slow, 20sec until it appears) 
- Launched "Spyder" from the navigator UI

Progress bar disappears after about 10-15sec. No traceback in the anaconda prompt.

I noticed that the Anaconda prompt by default launches me in the directory "C:\Local\Users\Firstname Surename\Docements" folder (i.e. it has a space.) So just to be sure I switched to "C:\" first and retried. Same result.

Just to confirm: I can launch Spyder from the anaconda promput by typing "Spyder" without issues.

**Comment by goanpeca** [comment]
@Bejoscha could you try to uninstall your current version, and reinstall the latest installers (https://www.anaconda.com/download/#windows), making sure to use installation as current user and use as install path something short like:

`C:\Local\Users\usernospaces\anaconda\`

**Comment by Bejoscha** [comment]
@goanpeca The installer is the one I've used. I, unfortunately, can not create a new user-account on my (domain)-machine due to enforced policies. The user-name _does_ have a space, hence I did install it (twice): Once for "all-users" at C:\Anaconda3  and once for "this user only" at C:\Anaconda3.

However, I've just realized that there _is_ a folder in **C:\user\User Name\\.anaconda** so it might still be the "space issue". 

However, creating a new user-account just for that is not an acceptable solution for me. If it can't be fixed, I just won't use the navigator.
